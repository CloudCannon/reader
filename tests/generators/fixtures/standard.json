{
	"data-config": {
		"authors": {
			"path": "tests/generators/fixtures/standard/data/authors.csv"
		},
		"fruit": {
			"path": "tests/generators/fixtures/standard/data/fruit"
		}
	},
	"collections-config": {
		"posts": {
			"path": "tests/generators/fixtures/standard/_posts",
			"output": true,
			"url": "hello/{title|slugify}"
		},
		"vegetables": {
			"path": "tests/generators/fixtures/standard/vegetables"
		}
	},
	"_comments": {},
	"_options": {},
	"_array_structures": {},
	"_select_data": {},
	"generator": {},
	"_source_editor": {},
	"paths": {
		"uploads": "assets/uploads"
	},
	"base-url": ""
}
