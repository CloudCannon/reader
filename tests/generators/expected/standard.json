{
  "data-config": {
    "authors": {
      "path": "tests/generators/fixtures/standard/data/authors.csv"
    },
    "fruit": {
      "path": "tests/generators/fixtures/standard/data/fruit"
    }
  },
  "collections-config": {
    "posts": {
      "path": "tests/generators/fixtures/standard/_posts",
      "output": true,
      "url": "hello/{title|slugify}"
    },
    "vegetables": {
      "path": "tests/generators/fixtures/standard/vegetables"
    }
  },
  "_comments": {},
  "_options": {},
  "_array_structures": {},
  "_select_data": {},
  "generator": {},
  "_source_editor": {},
  "paths": {
    "uploads": "assets/uploads"
  },
  "base-url": "",
  "time": "2000-11-22T00:00:00.000Z",
  "cloudcannon": {
    "name": "cloudcannon-reader",
    "version": "0.0.1"
  },
  "data": {
    "authors": [
      {
        "name": "Jim",
        "interest": "kayaks"
      },
      {
        "name": "Pam",
        "interest": "art"
      }
    ],
    "fruit": [
      {
        "name": "Apple",
        "rating": 5
      },
      {
        "name": "Banana",
        "rating": 4
      }
    ]
  },
  "collections": {
    "posts": [
      {
        "date": "2020-07-20T00:00:00.000Z",
        "title": "The process for direct sales",
        "categories": [
          "sales"
        ],
        "author_staff_member": "anna",
        "path": "tests/generators/fixtures/standard/_posts/2016-07-20-the-process-for-direct-sales.md",
        "collection": "posts",
        "url": "hello/the-process-for-direct-sales"
      },
      {
        "date": "2020-07-28T00:00:00.000Z",
        "title": "Effective upselling techniques",
        "categories": [
          "sales",
          "tips"
        ],
        "author_staff_member": "betty",
        "path": "tests/generators/fixtures/standard/_posts/2016-07-28-effective-upselling-techniques.md",
        "collection": "posts",
        "url": "hello/effective-upselling-techniques"
      },
      {
        "date": "2020-08-02T00:00:00.000Z",
        "title": "Sales effectiveness",
        "categories": [
          "sales",
          "tips"
        ],
        "author_staff_member": "gerald",
        "path": "tests/generators/fixtures/standard/_posts/2016-08-02-sales-effectiveness.md",
        "collection": "posts",
        "url": "hello/sales-effectiveness"
      },
      {
        "date": "2020-08-06T00:00:00.000Z",
        "title": "Definition of sales",
        "categories": [
          "sales"
        ],
        "author_staff_member": "james",
        "path": "tests/generators/fixtures/standard/_posts/2016-08-06-definition-of-sales.md",
        "collection": "posts",
        "url": "hello/definition-of-sales"
      },
      {
        "date": "2020-08-12T00:00:00.000Z",
        "title": "The history of marketing",
        "categories": [
          "marketing"
        ],
        "author_staff_member": "robin",
        "path": "tests/generators/fixtures/standard/_posts/2016-08-12-the-history-of-marketing.md",
        "collection": "posts",
        "url": "hello/the-history-of-marketing"
      },
      {
        "date": "2020-07-22T00:00:00.000Z",
        "title": "Egg",
        "categories": [
          "tips"
        ],
        "author_staff_member": "betty",
        "path": "tests/generators/fixtures/standard/_posts/nested/2020-07-22-egg.md",
        "collection": "posts",
        "url": "hello/egg"
      }
    ],
    "vegetables": [
      {
        "name": "Potato",
        "rating": 5,
        "path": "tests/generators/fixtures/standard/vegetables/potato.yml",
        "collection": "vegetables",
        "url": ""
      },
      {
        "name": "Yam",
        "rating": 4,
        "path": "tests/generators/fixtures/standard/vegetables/yam.yml",
        "collection": "vegetables",
        "url": ""
      }
    ]
  }
}
